graphql_compiler/tests/test_schema_generation.py

```diff
@@ -169,14 +169,14 @@ def test_parsed_superclasses_field(self):
         self.assertEqual({'Entity', ORIENTDB_BASE_VERTEX_CLASS_NAME},
                          schema_graph.get_inheritance_set('Entity'))
 
-    def test_parse_property(self):
+    def test_parsed_property(self):
```

Comment:
nit: is there a reason for `parse` -> `parsed`?

Reply:
For consistency with the other tests.